CMakeLists.txt

```diff
@@ -38,22 +38,42 @@ if(PALACE_WITH_STRUMPACK OR PALACE_WITH_MUMPS OR PALACE_WITH_ARPACK)
   enable_language(Fortran)
 endif()
 
-# Set a default installation location if none was provided
-if(NOT CMAKE_INSTALL_PREFIX OR CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-  message(STATUS "Setting CMAKE_INSTALL_PREFIX to '${CMAKE_BINARY_DIR}' as none was specified")
-  set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}" CACHE STRING "Install directory used by install()" FORCE)
-endif()
-
 # Set a default build type if none was provided
 if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
   message(STATUS "Setting CMAKE_BUILD_TYPE to 'Release' as none was specified")
-  set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Specifies the build type ('Debug' or 'Release', for example)" FORCE)
+  set(CMAKE_BUILD_TYPE "Release" CACHE STRING
+    "Specifies the build type ('Debug' or 'Release', for example)" FORCE
+  )
+endif()
+
+# Set a default installation location if none was provided
+if(NOT CMAKE_INSTALL_PREFIX OR CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  message(STATUS "Setting CMAKE_INSTALL_PREFIX to '${CMAKE_BINARY_DIR}' as none was specified")
+  set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}" CACHE STRING
+    "Install directory used by install()" FORCE
+  )
 endif()
 
 # Set a default for dependency library builds if none was provided
 if(NOT DEFINED BUILD_SHARED_LIBS)
   message(STATUS "Setting BUILD_SHARED_LIBS to 'OFF' as it was not specified")
-  set(BUILD_SHARED_LIBS OFF CACHE BOOL "Global flag to cause add_library() to create shared libraries if ON")
+  set(BUILD_SHARED_LIBS OFF CACHE BOOL
+    "Global flag to cause add_library() to create shared libraries if ON"
+  )
+endif()
+
+# Configure default RPATH for installed targets if not provided
+if(NOT DEFINED CMAKE_INSTALL_RPATH)
+  message(STATUS "Setting CMAKE_INSTALL_RPATH to '\${CMAKE_INSTALL_PREFIX}/lib;\${CMAKE_INSTALL_PREFIX}/lib64' as it was not specified")
```

Comment:
Based on the changes for line length elsewhere here, this might be split across lines with [\](https://stackoverflow.com/questions/7637539/how-to-split-strings-across-multiple-lines-in-cmake) to match.